src/components/GNB/profileTooltip.tsx

```
@@ -0,0 +1,48 @@
+import { useAuthStore } from '@/store/useAuthStore';
+import { useState } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useAuth } from '@/hooks/customs/useAuth';
+
+const ProfileTooltip = () => {
+	const { logoutUser } = useAuth();
+	const { image, setImage } = useAuthStore();
```

ESLint

[error] 'setImage' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)